source/development_process/configuring_git.rst

```diff
@@ -0,0 +1,71 @@
+Configuring Git
```

Comment:
We already have a page on configuring git that includes some of the things discussed here. Perhaps the two should be merged?